components/StyledComponents/StyledCard.js

```diff
@@ -0,0 +1,14 @@
+const { styled } = require("styled-components");
```

Comment:
```suggestion
import { styled } from "styled-components";
```
Get out, require! Shoo!